runtime/vm/lookupmethod.c

```diff
@@ -214,6 +214,12 @@ processMethod(J9VMThread * currentThread, UDATA lookupOptions, J9Method * method
 
 	if (((lookupOptions & J9_LOOK_STATIC) && ((modifiers & J9AccStatic) == 0))
 	|| ((lookupOptions & J9_LOOK_VIRTUAL) && (modifiers & J9AccStatic))
+#if JAVA_SPEC_VERSION < 11 && defined(J9VM_OPT_OPENJDK_METHODHANDLE)
+	/* Java 10- JVMS 6.5: if the resolved method is static or private, the invokeinterface instruction
+	 * throws an IncompatibleClassChangeError'
+	 */
+	|| ((lookupOptions & J9_LOOK_INTERFACE) && (lookupOptions & J9_LOOK_VIRTUAL) && (modifiers & J9AccPrivate))
+#endif /* JAVA_SPEC_VERSION < 11 && defined(J9VM_OPT_OPENJDK_METHODHANDLE) */
```

Comment:
For consistency since brackets are used in existing code.
```suggestion
#endif /* (JAVA_SPEC_VERSION < 11) && defined(J9VM_OPT_OPENJDK_METHODHANDLE) */
```